mazda_2017.dbc

```diff
@@ -770,6 +772,10 @@ CM_ SG_ 552 GEAR_BOX "0 P, 14 R, 1 though 6 D for speeds, 15 Shift";
 CM_ SG_ 540 HANDS_ON_STEER_WARN "0 no warning, b warning";
 CM_ SG_ 540 DISTANCE_SETTING "Radar distance 0: disabled, 1: 4 bars, 2: 3 bars, 3: 2 bars, 4: 1 bar";
 CM_ SG_ 1143 REAR_CT_ALERT "Rear Cross Traffic Alert";
+CM_ SG_ 1143 LEFT_BS1 "Left BlindSpot without blinker";
+CM_ SG_ 1143 RIGHT_BS1 "Right BlindSpot without blinker";
+CM_ SG_ 1143 LEFT_BS2 "Left Blindspot with left blinker activated - warning activated";
+CM_ SG_ 1143 RIGHT_BS2 "Right Blindspot with right blinker activated - warning activated";
```

Comment:
```suggestion
CM_ SG_ 1143 LEFT_BS_BLINKER "Left Blindspot with left blinker activated - warning activated";
CM_ SG_ 1143 RIGHT_BS_BLINKER "Right Blindspot with right blinker activated - warning activated";
```

seems like more appropriate names to me. would it make sense to combine into two 2 bit signals though since they are adjacent?

Reply:
We can, though I don't see any advantage either way

Reply:
Makes sense to combine them, it's likely just a 2 bit signal in the original DBC and we can check != 0 instead of ORing the two bits

![image](https://github.com/commaai/opendbc/assets/25857203/08c11284-db23-4e0e-acf9-77b6ee0d7eac)